test/codeu/chat/CreateInterestsTest.java

```diff
@@ -0,0 +1,5 @@
+package codeu.chat;
+
+public class CreateInterestsTest {
+
```

Comment:
will do. just need to find a way to create a UserContext object without actually starting the server with connection sources.